src/comparisons/elements/remove/modern.js

```diff
@@ -1 +1,10 @@
 el.remove();
+
+// one element:
+document.querySelector('el').remove();
+
+// multiple elements:
+for (const el
```

Comment:
Same dealio here from above:
```suggestion
for (const el of [...document.querySelectorAll(selector)]) {
```